src/txn/txn_ckpt.c

```diff
@@ -1501,10 +1501,10 @@ __txn_checkpoint_wrapper(WT_SESSION_IMPL *session, const char *cfg[])
 
     /*
      * FIXME-WT-11149: Some reading threads rely on the value of checkpoint running flag being
-     * published before the checkpoint generation number (set inside the checkpoint call below).
-     * Introduce a write barrier here to guarantee the right order.
+     * release written before the checkpoint generation number (set inside the checkpoint call
```

Comment:
may be just say "bing updated before the checkpoint generation number"

Reply:
b06cc6cec